helm/app-exporter/templates/service-monitor.yaml

```diff
@@ -9,9 +9,7 @@ metadata:
 spec:
   endpoints:
     - port: web
-      {{- if hasPrefix "cn-" .Values.provider.region }}
-      scrapeTimeout: 30s
```

Comment:
Previously `scrapeTimeout` was prolonged from 10 to 30 seconds only on Chinese installations. I see no reason to limit ourselves to such low values at all. We usually have plenty of Apps to scrape. This results in longer response times on some installations.

E.g. `helios`, which paged us because of requests to `/metrics` timing out in 10s three hundred times (300!) in the last 30 days. It takes roughly 30 seconds for the app-exporter there to respond with full metrics:
```
helios:thinkbook:~ time curl localhost:8000/metrics | tail -n5
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  216k    0  216k    0     0   6915      0 --:--:--  0:00:32 --:--:-- 57311
# HELP promhttp_metric_handler_requests_total Total number of scrapes by HTTP status code.
# TYPE promhttp_metric_handler_requests_total counter
promhttp_metric_handler_requests_total{code="200"} 20
promhttp_metric_handler_requests_total{code="500"} 0
promhttp_metric_handler_requests_total{code="503"} 0

________________________________________________________
Executed in   32.14 secs      fish           external
   usr time   23.20 millis    0.00 millis   23.20 millis
   sys time   24.98 millis    1.67 millis   23.31 millis

```

Reply:
The default scrape interval is 30s and Prometheus is currently crashing on `viking` due to 

```
level=error ts=2022-10-06T21:35:30.484Z caller=main.go:356 msg="Error loading config (--config.file=/etc/prometheus/config_out/prometheus.env.yaml)" err="parsing YAML file /etc/prometheus/config_out/prometheus.env.yaml: scrape timeout greater than scrape interval for scrape config with job name \"serviceMonitor/giantswarm/app-exporter/0\""
```

Sorry dude, need to reduce this to 30s to get some sleep. :/